src/services/aws/sqs_service.js

```diff
@@ -0,0 +1,34 @@
+import { serviceResponse } from '../../util';
```

Comment:
Reminder: this file shouldn't be here - sqs_service.js is removed. Should go away with your rebase on master.